src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -18,6 +18,7 @@
     "activationEvents": [
         "onWebviewPanel:razorReportIssue",
         "onDebugInitialConfigurations",
+        "onDebug",
```

Comment:
>onDebug [](start = 9, length = 7)

Don't you instead want: `onDebugResolve:blazorwasm`

Reply:
Yes, from my understanding both work but `onDebugResolve:blazorwasm` is the more specific handler of the two. I've updated this now.